docs/manual/appendix/JVMMemoryOnDev.md

```diff
@@ -0,0 +1,32 @@
+# Increase Memory in SBT and Maven
+
+Lagom in dev mode starts all your services plus some internal Lagom services and in a single JVM. This can cause `OutOfMemoryError`s depending on your JVM setup. It is possible to start maven and sbt with increased memory.
```

Comment:
Capitalize Maven here.  Also, the "and" from "and in a single JVM" can be removed.